pkg/backend/filestate/backend.go

```diff
@@ -538,3 +538,19 @@ func (b *localBackend) getLocalStacks() ([]tokens.QName, error) {
 
 	return stacks, nil
 }
+
+// GetStackTags fetches the stack's existing tags.
+func (b *localBackend) GetStackTags(ctx context.Context,
+	stackRef backend.StackReference) (map[apitype.StackTagName]string, error) {
+
+	// The local backend does not currently persist tags.
+	return nil, nil
+}
+
+// UpdateStackTags updates the stacks's tags, replacing all existing tags.
+func (b *localBackend) UpdateStackTags(ctx context.Context,
+	stackRef backend.StackReference, tags map[apitype.StackTagName]string) error {
+
+	// The local backend does not currently persist tags.
```

Comment:
If we do end up wanting to implement this for the local backend, I was thinking we'd store these in a file in the same directory as the stack checkpoints, e.g.

```go
func (b *localBackend) stackTagsPath(stack tokens.QName) string {
	contract.Require(stack != "", "stack")
	return filepath.Join(b.StateDir(), workspace.StackDir, fsutil.QnamePath(stack)+".tags.json")
}
```